web/static/js/pages/profile.js

```diff
@@ -860,4 +842,31 @@ function beatmapLink(type, id) {
 	if (type == "s")
 		return "<a href='/s/" + id + "'>" + id + '</a>';
 	return "<a href='/b/" + id + "'>" + id + '</a>';
+}
+
+function checkRelaxMania(mode, rx) {
+	if (rx == 1) {
+		for (i = 0; i <= 3; i++) {
+			$(`.simple-banner-swtich[data-mode='${i}']`).removeClass('disabled')
+		}
+		$(".simple-banner-swtich[data-mode='3']").addClass('disabled')
+	} else if (rx == 2) {
+		for (i = 1; i <= 3; i++) {
+			$(`.simple-banner-swtich[data-mode='${i}']`).addClass('disabled')
+		}
+	} else {
+		for (i = 0; i <= 3; i++) {
+			$(`.simple-banner-swtich[data-mode='${i}']`).removeClass('disabled')
+		}
+	}
+
+	if (mode === 3) {
+		$(".simple-banner-swtich[data-rx='1']").addClass('disabled')
+		$(".simple-banner-swtich[data-rx='2']").addClass('disabled')
+	} else if (mode === 1 || mode === 2) {
+		$(".simple-banner-swtich[data-rx='2']").addClass('disabled')
+	} else {
+		$(".simple-banner-swtich[data-rx='1']").removeClass('disabled')
+		$(".simple-banner-swtich[data-rx='2']").removeClass('disabled')
+	}
 }
```

Comment:
@lenforiee hey, why did this get reverted?

pretty sure we were using `toggleModeAvailability`, right? or am i misremembering

Reply:
`swtich` what the hell

Reply:
ah yes found the issue - toggleModeAvailability was defined in dist.min.js but not in the other pages (like clan.js)